ndsi/formatter.py

```diff
@@ -249,3 +244,7 @@ class _IMUDataFormatter_V4(IMUDataFormatter):
     def decode_msg(self, data_msg: DataMessage) -> IMUValue:
         content = np.frombuffer(data_msg.body, dtype=self.CONTENT_DTYPE).view(np.recarray)
         return IMUValue(*content)
+
+
+class _IMUDataFormatter_V4(_IMUDataFormatter_V3):
+    pass
```

Comment:
I was using `NDSIv4` as a guide: https://github.com/pupil-labs/pyndsi/blob/NDSIv4/ndsi/sensor.pyx#L301-L307

It would be great to get `IMU` samples for both `v3` and `v4` to make sure everything works as expected.

Reply:
Updated